sktime/dists_kernels/compose_tab_to_panel.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""
+Composers that create panel pairwise transformers from table pairwise transformers
+"""
+
+__author__ = ["fkiraly"]
+
+import numpy as np
+
+from sktime.dists_kernels._base import BasePairwiseTransformerPanel
+
+
+class AggrDist(BasePairwiseTransformerPanel):
+    """
+    pairwise distance based on aggregation function applied to sample-wise distance
+        example: AggrDist(ScipyDist()) is mean Euclidean distance between series
```

Comment:
Potentially major but I don't know:
* Does `SciPyDist()` default to Euclidean distance? Or is this a typo? Or something else?

Reply:
@fkiraly Just to confirm, the resolution to this conversation is that this is not an issue and does not require change? (which, to be clear, I am ok with. but just checking that was your intention)

Reply:
it says in the docstring, default='euclidean', so I thought it was not an issue.

Reply:
Just for clarity, my (minor) concern is that this default setting is only explicit in the docstring for ScipyDist, not in the docstring for AggrDist. My preference would be something like `example: AggrDist(ScipyDist(metric = 'euclidean')) is mean Euclidean...`